src/modules/app-configuration/fallback-app-config.ts

```diff
@@ -0,0 +1,22 @@
+import { AppConfig } from "./app-config";
+import { AppConfigContainer } from "./app-config-container";
+import { ChannelFragment, ShopInfoFragment } from "../../../generated/graphql";
+
+/**
+ * TODO Test
+ */
+export const FallbackAppConfig = {
+  createFallbackConfigFromExistingShopAndChannels(
+    channels: ChannelFragment[],
+    shopAppConfiguration: ShopInfoFragment | null
+  ) {
+    return (channels ?? []).reduce<AppConfig>(
+      (state, channel) => {
+        return AppConfigContainer.setChannelAppConfiguration(state)(channel.slug)({
+          active: "",
+        });
+      },
+      { shopConfigPerChannel: {} }
```

Comment:
maybe its better to explicitly set `null` when config was never set. Its easier to detect (`if(! shopConfigPerChannel)`) and we will likely avoid bugs